lib/seahorn/BmcPass.cc

```diff
@@ -87,23 +99,25 @@ class BmcPass : public llvm::ModulePass {
   }
 
   void getAnalysisUsage(AnalysisUsage &AU) const {
-#ifdef HAVE_CRAB_LLVM
+#ifdef HAVE_CRAB_LLVM    
```

Comment:
The logic for `getAnalysisUsage` seems confusing. Maybe it is only in the diff. 
What is included and when. Code that is commented should be removed. Why do we want to lower constant expressions at this point? Seems like a hack that has outlived its purpose.